src/pages/components/parallax.mdx

```diff
@@ -3,7 +3,55 @@
 ## Overview
```

Comment:
Lets add a h1 that says `Parallax`

Reply:
Should be there, but I think it was just cut out of the diff.